src/encoded/static/components/search.js

```diff
@@ -1300,6 +1305,7 @@ ResultTable.propTypes = {
     searchBase: PropTypes.string,
     onChange: PropTypes.func.isRequired,
     currentRegion: PropTypes.func,
+    region: PropTypes.string,
```

Comment:
eslint error. AirBnB requires a `defaultProps` if you don’t have `isRequired`.

Reply:
Hopefully I have addressed these issues, Forrest.